lib/config/types.ts

```diff
@@ -194,6 +194,7 @@ export type ExecutionMode = 'branch' | 'update';
 
 export interface PostUpgradeTasks {
   commands?: string[];
+  dataFileTemplate?: string;
```

Comment:
I think we could grant ourselves more flexibility if we make this an array with multiple string for multiple files. Would that make sense?

Reply:
It has been discussed in https://github.com/renovatebot/renovate/discussions/36040 and the resolution is to use a single file.